Description: Add etc/graphite to sys.path to load config from /etc/graphite
Author: Jonas Genannt <jonas.genannt@capi2name.de>
Forwarded: not-needed

--- a/webapp/graphite/settings.py
+++ b/webapp/graphite/settings.py
@@ -20,6 +20,9 @@
 from os.path import abspath, dirname, join
 from warnings import warn
 
+# Debian add etc/graphite into path
+sys.path.append('/etc/graphite')
+
 try:
   import rrdtool
 except ImportError:
@@ -128,7 +131,7 @@
 
 ## Load our local_settings
 try:
-  from graphite.local_settings import *
+  from local_settings import *
 except ImportError:
   print >> sys.stderr, "Could not import graphite.local_settings, using defaults!"
 
